<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DealEditorUI</class>
 <widget class="QDialog" name="DealEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choisissez les cartes pour chaque joueur. Les cartes restantes dans la colonne de gauche constituent le chien.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="mainCardList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="toSouth">
         <property name="text">
          <string>Sud</string>
         </property>
         <property name="icon">
          <iconset resource="../data/data.qrc">
           <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toWest">
         <property name="text">
          <string>Ouest</string>
         </property>
         <property name="icon">
          <iconset resource="../data/data.qrc">
           <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="southList"/>
       </item>
       <item>
        <widget class="QListWidget" name="westList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="toNorth">
         <property name="text">
          <string>Nord</string>
         </property>
         <property name="icon">
          <iconset resource="../data/data.qrc">
           <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toEast">
         <property name="text">
          <string>Est</string>
         </property>
         <property name="icon">
          <iconset resource="../data/data.qrc">
           <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="northList"/>
       </item>
       <item>
        <widget class="QListWidget" name="eastList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ouvrir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Sauvegarder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Fermer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../data/data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>DealEditorUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
